metrics/factory.go

```diff
@@ -75,14 +75,15 @@ func Exists(domain string) bool {
 // Make makes a metric collector for the domain using a previously registered factory.
 //
 // See types in the blip package for more details.
-func Make(domain string, args blip.CollectorFactoryArgs) (blip.Collector, error) {
+func Make(domain string, args blip.CollectorFactoryArgs, frequency int) (blip.Collector, error) {
```

Comment:
We can get rid of `frequency` and all the new plumbing too, right?

Reply:
Shoot, yes.

Reply:
All cleaned up.